homeassistant/components/tibber/config_flow.py

```diff
@@ -0,0 +1,65 @@
+"""Adds config flow for Tibber integration."""
+import asyncio
+import logging
+
+import aiohttp
+import tibber
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_ACCESS_TOKEN
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema({vol.Required(CONF_ACCESS_TOKEN): str})
+
+
+class TibberConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Tibber integration."""
+
+    VERSION = 1
+
+    async def async_step_import(self, import_info):
+        """Set the config entry up from yaml."""
+        return await self.async_step_user(import_info)
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+        if user_input is None:
+            return self.async_show_form(
+                step_id="user", data_schema=DATA_SCHEMA, errors={},
+            )
+
+        access_token = user_input[CONF_ACCESS_TOKEN].replace(" ", "")
+
+        tibber_connection = tibber.Tibber(
+            access_token=access_token, websession=async_get_clientsession(self.hass),
+        )
+
+        errors = {}
+
+        try:
+            await tibber_connection.update_info()
+        except asyncio.TimeoutError:
+            errors[CONF_ACCESS_TOKEN] = "timeout"
+        except aiohttp.ClientError:
+            errors[CONF_ACCESS_TOKEN] = "connection_error"
+        except tibber.InvalidLogin:
+            errors[CONF_ACCESS_TOKEN] = "invalid_access_token"
+
+        if errors:
+            return self.async_show_form(
+                step_id="user", data_schema=DATA_SCHEMA, errors=errors,
+            )
+
+        unique_id = "tibber"
```

Comment:
That's not very unique and will prevent Home Assistant from configuring two accounts. Can't we use `viewer.login` ? Otherwise we should skip the unique ID.

Reply:
Thanks for the review.

The intention was to only allow one account.
You can have several homes connected to one account, so do not see the need for several accounts.
I am not sure how to handle several accounts for the notify platform?

Reply:
Btw, I was not aware of viewer.login. Is that a predefined view?

Reply:
viewer.login is what I looked up in the Tibber API explorer https://developer.tibber.com/explorer :) 

We should really fix the notification integration 🙄 I think that you can actually manage multiple accounts in notify, by having the platform support targets. But that is out of scope for this PR, so just 1 config entry is fine. However, still do not abuse our unique IDs for this or else you're in trouble in the future.

Just do this.

```python
if self._async_current_entries():
    return self.async_abort(reason='already_configured')
```